.github/workflows/java.yml

```diff
@@ -19,7 +19,6 @@ jobs:
     - name: Install nightly rust
       uses: actions-rs/toolchain@v1
       with:
-        toolchain: nightly
```

Comment:
Oh, that's great that that just works! I didn't even check.

Reply:
It's documented as working for the action, but isn't finding the `rust-toolchain` file for some reason, further debugging required.

Reply:
Looks like we've just got the steps in the wrong order—checkout first, then install. We haven't had to do that before now.

Reply:
I wasn't able to get it to read a rust-toolchain even with a simple repo so I opened an issue with actions-rs https://github.com/actions-rs/toolchain/issues/123. I'm going to wait to hear back from them, or if nothing then I'll try the approach you used for node.